desktop/main.js

```diff
@@ -201,15 +198,17 @@ const mainWindow = (() => {
 
             // When the user clicks a link that has target="_blank" (which is all external links)
             // open the default browser instead of a new electron window
-            browserWindow.webContents.on('new-window', (e, url) => {
-                // make sure local urls stay in electron perimeter
-                if (url.substr(0, 'file://'.length) === 'file://') {
-                    return;
+            browserWindow.webContents.setWindowOpenHandler(({url}) => {
+                const denial = {action: 'deny'};
+
+                // Make sure local urls stay in electron perimeter
+                if (url.substr(0, 'file://'.length).toLowerCase() === 'file://') {
```

Comment:
Note: it used to be possible to open apps and files (bypassing the blacklist we had here), simply by using a URL like `File://` or `fiLE://`.